pkgs/applications/networking/instant-messengers/blink/default.nix

```
@@ -1,68 +1,93 @@
-{ lib, fetchdarcs, python2Packages, libvncserver, zlib
-, gnutls, libvpx, makeDesktopItem, mkDerivationWith }:
-
-mkDerivationWith python2Packages.buildPythonApplication rec {
+{ lib, python37Packages, libvncserver, xorg, makeDesktopItem
```

generally it's easier to override the interpreter. The package set can be referenced through `python.pkgs`
```suggestion
{ lib, python37, libvncserver, xorg, makeDesktopItem
```

---

Thanks. I ended up bumping up to `python3`, but I will use the interpreter instead of the packages alias.